recipes/pyhunspell/build.sh

```diff
@@ -1,11 +0,0 @@
-
```

Comment:
I don't know enough about Java to review the rest but please don't delete other recipes from this folder. Only yours should change. The bot will take care of the rest.

Reply:
But I removed those files because I created a new branch from another branch where I was creating another recipe and I forgot to checkout first to master. Even in that case I need to let the wrong files from the old recipe? What should I do now?